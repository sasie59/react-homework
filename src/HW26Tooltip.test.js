import React from 'react';
import { render } from '@testing-library/react';
import HW from './HW26Tooltip';

test('should ', () => {
  const {container} = render(<HW />);
  expect(container).toMatchSnapshot();
});
