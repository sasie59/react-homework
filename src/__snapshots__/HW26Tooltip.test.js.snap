// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should  1`] = `
<div>
  <div>
    <br />
    <br />
    <br />
    <br />
    <br />
    <span
      class="tooltip-wrapper"
    >
      <button
        class="btn"
      >
        top
      </button>
      <div
        class="tooltip top"
      >
        我在上面
      </div>
    </span>
    <br />
    <br />
    <span
      class="tooltip-wrapper"
    >
      <button
        class="btn"
      >
        right
      </button>
      <div
        class="tooltip right"
      >
        我在右邊
      </div>
    </span>
    <br />
    <br />
    <span
      class="tooltip-wrapper"
    >
      <button
        class="btn"
      >
        left
      </button>
      <div
        class="tooltip left"
      >
        我在左邊
      </div>
    </span>
    <br />
    <br />
    <span
      class="tooltip-wrapper"
    >
      <button
        class="btn"
      >
        bottom
      </button>
      <div
        class="tooltip bottom"
      >
        我在下面
      </div>
    </span>
    <br />
    <br />
    <br />
    Lorem
    <span
      class="tooltip-wrapper"
    >
      <span
        style="color: red;"
      >
        ipsum
      </span>
      <div
        class="tooltip top"
      >
        我在上面
      </div>
    </span>
    dolor sit amet consectetur adipisicing elit. Temporibus nostrum atque similique voluptatum laborum soluta ad. Repellendus laborum ratione dignissimos aut quas necessitatibus ut officiis fuga quis odit, labore
    <span
      class="tooltip-wrapper"
    >
      <span
        style="color: red;"
      >
        vero.
      </span>
      <div
        class="tooltip bottom"
      >
        我在下面
      </div>
    </span>
  </div>
</div>
`;
